src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java

```diff
@@ -123,11 +123,34 @@ public class LaDigitalAssister extends FlowInputs {
 
   private String schoolInd;
 
-  private String students;
+  @NotEmpty(message="{error.missing-general}")
+  private List<String> students;
+
+  @NotBlank(message="{error.missing-general}")
+  @DynamicField
+  private String schoolName;
+
+  @NotBlank(message="{error.missing-general}")
+  @DynamicField
+  private String schoolEnrollmentLevel;
 
   private String pregnancyInd;
 
-  private String pregnancies;
+  @NotEmpty(message="{error.missing-general}")
+  private List<String> pregnancies;
+
+//  With dynamic fields and the date template, the data is stored as pregnancyDueDate<wildcard><uuid>Day
+//  @NotBlank(message="{error.missing-general}")
+//  @DynamicField
+  private String pregnancyDueDateDay;
+
+//  @NotBlank(message="{error.missing-general}")
+//  @DynamicField
+  private String pregnancyDueDateMonth;
+
+//  @NotBlank(message="{error.missing-general}")
+//  @DynamicField
+  private String pregnancyDueDateYear;
```

Comment:
I'm noticing that the field names used in the templates don't align with what is here, and maybe that is why they are not working?  `name="pregnancyDueDate_wildcard_9384801b-95ab-4907-aab6-376b634d4a78Year"`

Reply:
I see why below.  The fragments are definitely not prepare to work with dynamic fields, but I think we can definitely tweak them to fit our needs.